src/pages/inward-transfer/forms/InwardTransferForm.js

```diff
@@ -0,0 +1,1093 @@
+    receiver_address1: '',
+    receiver_address2: '',
+    receiver_bank: '',
+    receiver_bankBranch: '',
+    receiver_ttNo: '',
+    paymentMode: '',
+    paymentBank: '',
+    commissionType: '',
+    commissionAgent: '',
+    commissionReceiver: '',
+    expiryDate: null,
+    sourceOfIncome: '',
+    purposeOfTransfer: ''
+  })
+
+  const { maxAccess } = useDocumentType({
+    functionId: SystemFunction.InwardTransfer,
+    access: access,
+    enabled: !recordId
+  })
+
+  const { formik } = useForm({
+    maxAccess,
+    initialValues,
+    enableReinitialize: true,
+    validateOnChange: true,
+    validationSchema: yup.object({
+      date: yup.date().required(),
+      corId: yup.string().required(),
+      currencyId: yup.string().required(),
+      amount: yup.number().required(),
+      transferType: yup.string().required(),
+      faxNo: transferType == '1' ? yup.string().required() : yup.string().notRequired(),
+      sender_firstName: yup.string().required(),
+      sender_lastName: yup.string().required(),
+      sender_nationalityId: yup.string().required(),
+      sender_countryId: yup.string().required(),
+      receiver_type: yup.string().required(),
+      receiver_firstName: yup.string().required(),
+      receiver_lastName: yup.string().required(),
+      receiver_payoutType: yup.string().required(),
+      commissionAgent: yup.number().required(),
+      commissionReceiver: yup.number().required()
+    }),
+    onSubmit: async values => {
+      const copy = { ...formik.values }
+      copy.date = formatDateToApi(copy?.date)
+      copy.wip = copy?.wip === '' ? 1 : copy?.wip
+      copy.exRate = copy?.exRate === '' ? 1 : copy?.exRate
+      copy.status = copy?.status === null ? 1 : copy?.status
+      copy.baseAmount = copy?.baseAmount === '' ? copy?.amount : copy?.baseAmount
+      copy.rateCalcMethod = copy?.rateCalcMethod === '' ? 1 : copy?.rateCalcMethod
+      copy.sender_idIssueDate = copy.sender_idIssueDate ? formatDateToApi(copy?.sender_idIssueDate) : null
+      copy.sender_idExpiryDate = copy.sender_idExpiryDate ? formatDateToApi(copy?.sender_idExpiryDate) : null
+      copy.receiver_idIssueDate = copy.receiver_idIssueDate ? formatDateToApi(copy?.receiver_idIssueDate) : null
+      copy.receiver_idExpiryDate = copy.receiver_idExpiryDate ? formatDateToApi(copy?.receiver_idExpiryDate) : null
+      copy.expiryDate = copy.expiryDate ? formatDateToApi(copy?.expiryDate) : null
+
+      const res = await postRequest({
+        extension: RemittanceOutwardsRepository.InwardsTransfer.set,
+        record: JSON.stringify(copy)
+      })
+      if (res.recordId) {
+        formik.setFieldValue('recordId', res.recordId)
+        setEditMode(true)
+        invalidate()
+
+        const res2 = await getRequest({
+          extension: RemittanceOutwardsRepository.InwardsTransfer.get,
+          parameters: `_recordId=${res.recordId}`
+        })
+        formik.setFieldValue('reference', res2.record.reference)
```

Comment:
remove 
        formik.setFieldValue('recordId', res.recordId)
        formik.setFieldValue('reference', res2.record.reference)
        formik.setFieldValue('status', res2.record.status)
       
and update all formik fields values from res2 response